src/cli/utils.ts

```diff
@@ -1,118 +0,0 @@
-import type { Result, ResultMap } from '../types';
-import { ResultsByGroupId, Results } from './types';
-
-/* eslint-disable no-console */
-export const debug = (...args: string[]) => console.warn(...args);
-export const stdout = (...args: string[]) => console.log(...args);
-
-export const usage = () =>
-  debug(`Usage:
-sb-perf <directory> [...<directory>]
-
-Example
-sb-perf results-directory > output-file.csv
-# OR
-sb-perf ABTestDirectory OtherDirectory > output-file.csv
-`);
-
-const getTaskValue = (result: Result) => {
-  if ('averageMs' in result) {
-    return result.averageMs;
-  }
-
-  if ('value' in result) {
-    return Number(result.value);
-  }
-
-  return null;
-};
-
-export const convertToTaskValueMap = (resultMap: ResultMap) => {
-  return Object.values(resultMap).reduce((acc, result) => {
-    return { ...acc, [result.taskName]: [getTaskValue(result)] };
-  }, {});
-};
-
-export const combineTaskResultsByGroupId = (
-  results: ResultsByGroupId,
-  [groupId, taskValueMap]: [string, Results],
-) => {
-  Object.entries(taskValueMap).forEach(([taskName, value]) => {
-    results[groupId] = {
-      ...results[groupId],
-      [taskName]:
-        results[groupId] && results[groupId][taskName]
-          ? results[groupId][taskName].concat(value)
-          : value,
-    };
-  });
-
-  return results;
-};
-
-export const median = (numbers: number[]) => {
-  const sorted = numbers.slice().sort((a, b) => a - b);
-  const middle = Math.floor(sorted.length / 2);
-
-  if (sorted.length % 2 === 0) {
-    return (sorted[middle - 1] + sorted[middle]) / 2;
-  }
-
-  return sorted[middle];
-};
-
-export const performCalculations = (data: Results) => {
-  // extracts the number of samples in the dataset
-  const numResults = Object.values(data)[0].length;
-  return Object.entries(data).map(([key, values]) => ({
-    key,
-    numResults,
-    samples: values.join(','),
-    minValue: Math.min(...values),
-    maxValue: Math.max(...values),
-    meanValue: values.reduce((acc, curr) => acc + curr, 0) / numResults,
-    medianValue: median(values),
-  }));
-};
-
-export type Row = ReturnType<typeof performCalculations>[number];
-
-/**
- * Turns the row output into a csv
- */
-export const printCSV = (rows: Row[]) => {
-  // extracts the number of samples the dataset
-  const numResults = rows[0].numResults;
-  stdout(
-    `type,${Array.from({ length: numResults })
-      .map((_, i) => `#${i + 1}`)
-      .join(',')},min,max,mean,median`,
-  );
-  rows.forEach(({ key, samples, minValue, maxValue, meanValue, medianValue }) => {
-    stdout(`${key},${samples},${minValue},${maxValue},${meanValue},${medianValue}`);
-  });
-};
-
-const padded = (padding: number) => (str: string | number) => `${str}`.padEnd(padding);
-const paddedKey = padded(50);
-const paddedValue = padded(20);
-
-export const printCSVSummary = (resultNames: string[], results: Row[][]) => {
```

Comment:
I'm just wondering if we should keep this for glance value or does it complicate things?

Reply:
hmm i feel like it has less value if the comparison data is not there though? maybe we'll bring it back if we want to iterate on it and include the comparison data?